feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/domain/nft/search/NftSearchInteractor.kt

```diff
@@ -21,30 +23,36 @@ class NftSearchInteractor @Inject constructor(
 
     fun sendNftSearch(queryFlow: Flow<String>): Flow<Map<Chain, List<SendNftListItem>>> {
         return combine(getUserNfts(), queryFlow) { nfts, query ->
-            val supportedNfts = nfts
-                .filter { nftRepository.isNftTypeSupportedForSend(it.type) }
-
-            mapNftToNftDetails(supportedNfts)
-                .map(::mapNftDetailsToListItem)
-                .filterByQuery(query)
+            nfts.filterByQuery(query)
                 .groupBy { nftDetails -> nftDetails.chain }
         }
     }
 
-    private fun getUserNfts(): Flow<List<Nft>> {
+    private fun getUserNfts(): Flow<List<SendNftListItem>> {
```

Comment:
Maybe worth naming it something like `getUserSendableNfts` given that we only return sendable nfts here?

Reply:
Done